sycl/include/sycl/ext/oneapi/kernel_properties/properties.hpp

```diff
@@ -246,8 +275,13 @@ template <forward_progress_guarantee Guarantee,
 struct property_value<
     sub_group_progress_key,
     std::integral_constant<forward_progress_guarantee, Guarantee>,
-    std::integral_constant<execution_scope, CoordinationScope>> {
-  using key_t = work_group_progress_key;
+    std::integral_constant<execution_scope, CoordinationScope>>
+    : detail::property_base<
+          property_value<
+              sub_group_progress_key,
```

Comment:
This and the next chunk are bug-fixes for typos on the left side when wrong type was used in the `key_t` type aliases - all three `*progress_key`s used `work_group_progress_key` instead of individual keys.